### ISSUE TITLE
Intermittent errors in ibm_container_cluster_config (slice bounds out of range [8:0] - race condition?)

### ISSUE THREAD

**Issue by vburckhardt** [IBM-Cloud/terraform-provider-ibm#2743]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version

```
terraform -v
Terraform v0.15.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/helm v2.1.2
+ provider registry.terraform.io/hashicorp/kubernetes v2.3.1
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/time v0.7.1
+ provider registry.terraform.io/ibm-cloud/ibm v1.26.0
```

### Affected Resource(s)

data "ibm_container_cluster_config"

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
##############################################################################
# Login to cluster
##############################################################################

data "ibm_container_cluster_config" "cluster_config" {
  cluster_name_id = var.cluster_id
}

##############################################################################
# Config providers
##############################################################################

provider "ibm" {
  ibmcloud_api_key = var.ibmcloud_api_key
}

provider "helm" {
  kubernetes {
    host                   = data.ibm_container_cluster_config.cluster_config.host
    token                  = data.ibm_container_cluster_config.cluster_config.token
    cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate
  }
}

provider "kubernetes" {
  host                   = data.ibm_container_cluster_config.cluster_config.host
  token                  = data.ibm_container_cluster_config.cluster_config.token
  cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate
}
```

### Debug Output

Git with full trace: https://gist.github.com/vburckhardt/49b914b363383a4113114970cbb18efe

### Actual Behavior

Intermittent errors:
`Error downloading the cluster config [c2skm3gd0nm0nnpu9uj0]: Could not login to openshift account runtime error: slice bounds out of range [8:0]`

Re-running a few times the command get past this - it does seem to be some race condition?

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

VPC OpenShift clusters. Reproducable on various machines.

### References

* #0000

**Comment by hkantare** [comment]
I think this is some kind of intermittent or race condition where the master url of the cluster is not reachable...
We are planning to add some retries on such race condition...
This is PR which is in development to handle retries
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2774

We are planning to make this fix available in next release planned in couple of days(v1.27.0)

**Comment by kavya498** [comment]
Available in the [1.27.0](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.27.0)

**Comment by kavya498** [comment]
Closing this issue..Thanks
